### DISCUSSION TITLE
How do I automatically see function arguments when I start writing a python function call?

### DISCUSSION THREAD

**Discussion by armanschwarz** [hrsh7th/nvim-cmp#1069] [Q&A] [answered]
The video on the home page of the repo has a really nifty looking feature where the user starts typing "table.insert" and in italics the rest of the function parameter types are suggested. How can I get this in python?

I currently have nvim-cmp working with lspconfig/pyright, using basically just the "Recommended Configuration" except I have configured lspconfig like this:

```
    -- Setup lspconfig.                                                                  
    local capabilities = require('cmp_nvim_lsp').update_capabilities(vim.lsp.protocol.make_client_capabilities())
    require('lspconfig')['pyright'].setup {                                             
      capabilities = capabilities                                                      
    }
```

When I start typing module and function names I see suggestions, but as soon as I type the open paranthesis character "(" I don't see any text popping up.

**Comment by fitrh** [comment, marked as answer]
I don't know what you really want, but I can suggest you to do the following:

1. Enable `experimental = { ghost_text = true }`
2. Setup [lsp_signature.nvim](https://github.com/ray-x/lsp_signature.nvim)
